api/v1/models/Hacker.js

```diff
@@ -1,11 +1,13 @@
 /* jshint esversion: 6 */
 
-var registration = require('../utils/registration');
+var utils = require('../utils');
+var registration = utils.registration;
 var _ = require('lodash');
 
 var OMITTED_ATTRIBUTES = [];
```

Comment:
Caps are generally reserved for constants, so it's probably a good idea not to lower case these.